drivers/storage/ide/uniata/id_probe.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Assumed unwanted change?

Reply:
Could be a change in the ByteOrderMark? We do want our files to be UTF8 without BOM.
Please make sure to only edit with editors that won't break the format. Notepad++ is fine. Notepad is not,

Reply:
Strange, removed :+1: